CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Next release
 
+- test: Deleting tests for wallets
```

Comment:
you didn't deleted any tests in the end. Did you?

Reply:
I deleted the tests related to the wallets that you told me the other day.

Reply:
after adding them yourself. But in the final PR there is no deletion. So don't include it in the changelog